openwisp_ipam/base/models.py

```diff
@@ -188,11 +188,17 @@ def import_csv(self, file):
             reader = iter(row)
         else:
             reader = csv.reader(StringIO(file.read().decode('utf-8')), delimiter=',')
-        subnet = self._read_subnet_data(reader)
+        subnet = self._read_subnet_data(reader, *user)
         next(reader)
         next(reader)
         self._read_ipaddress_data(reader, subnet)
 
+    def validate_permission(self, org, user):
+        if not (user.is_superuser or user.is_manager(org)):
+            raise PermissionDenied(
+                "User does not have access to the specified organization"
```

Comment:
```suggestion
                'User does not have access to the specified organization'
```

Reply:
wait, why are you doing these changes here? 
I am not sure this is the right place to do this check, why is this needed at all, can you explain?

Reply:
Now that I look at it, this is not a good way to implement what I tried to do. Basically, a user could still access subnets from other organizations by importing/exporting CSV files (which have an organization field in them). I will try to make a reusable class for this too.